diff --git a/dist/router.cjs.js b/dist/router.cjs.js
index 0213beedd90e41d2aa38993137a9dcb79f60d1df..83b7d96b88680fac9626536fa3a5bf01297e5644 100644
--- a/dist/router.cjs.js
+++ b/dist/router.cjs.js
@@ -3936,7 +3936,7 @@ function createStaticHandler(routes, opts) {
       let results = await callDataStrategy("action", request, [actionMatch], matches, isRouteRequest, requestContext, unstable_dataStrategy);
       result = results[0];
       if (request.signal.aborted) {
-        throwStaticHandlerAbortedError(request, isRouteRequest, future);
+        //throwStaticHandlerAbortedError(request, isRouteRequest, future);
       }
     }
     if (isRedirectResult(result)) {
@@ -4051,7 +4051,7 @@ function createStaticHandler(routes, opts) {
     }
     let results = await callDataStrategy("loader", request, matchesToLoad, matches, isRouteRequest, requestContext, unstable_dataStrategy);
     if (request.signal.aborted) {
-      throwStaticHandlerAbortedError(request, isRouteRequest, future);
+      //throwStaticHandlerAbortedError(request, isRouteRequest, future);
     }
 
     // Process and commit output from loaders
