diff --git a/src/ts/store.ts b/src/ts/store.ts
--- a/src/ts/store.ts
+++ b/src/ts/store.ts
@@ -962,7 +962,6 @@ else {
 
 /** @private */
 function initCDVPurchase() {
-    console.log('Create CdvPurchase...');
     const oldStore = window.CdvPurchase?.store;
     window.CdvPurchase = CdvPurchase;
     if (oldStore) {
diff --git a/www/store.js b/www/store.js
--- a/www/store.js
+++ b/www/store.js
@@ -2491,7 +2491,6 @@ else {
 /** @private */
 function initCDVPurchase() {
     var _a;
-    console.log('Create CdvPurchase...');
     const oldStore = (_a = window.CdvPurchase) === null || _a === void 0 ? void 0 : _a.store;
     window.CdvPurchase = CdvPurchase;
     if (oldStore) {
