diff --git a/lib/rules/no-extraneous-dependencies.js b/lib/rules/no-extraneous-dependencies.js
index 220e15a25afc9554cd055c1bc5f2405948ba86ef..d8d412b3d23a11c15743297fc353ff2584e0090d 100644
--- a/lib/rules/no-extraneous-dependencies.js
+++ b/lib/rules/no-extraneous-dependencies.js
@@ -217,6 +217,9 @@ function reportIfMissing(context, deps, depsOptions, node, name) {
   // test the real name from the resolved package.json
   // if not aliased imports (alias/react for example), importPackageName can be misinterpreted
   var realPackageName = getModuleRealName(resolved);
+  if (realPackageName?.startsWith('prisma-client-')) {
+    return;
+  }
   if (realPackageName && realPackageName !== importPackageName) {
     declarationStatus = checkDependencyDeclaration(deps, realPackageName, declarationStatus);
 
