diff --git a/src/index.js b/src/index.js
index 31e3666ef5e57ca836a5d9519b713ba61be9ea2c..3f2b4ea25f033d3edd45c1ebad4dd03ba1f4c40b 100644
--- a/src/index.js
+++ b/src/index.js
@@ -13,7 +13,6 @@
  */
 /* eslint-disable no-underscore-dangle */
 const kebabCase = require('lodash/kebabCase');
-const merge = require('lodash/merge');
 const path = require('path');
 const Chalk = require('chalk').Instance;
 const { diffImageToSnapshot, runDiffImageToSnapshot } = require('./diff-snapshot');
@@ -32,17 +31,11 @@ function toBuffer(data) {
   return Buffer.from(data);
 }
 
-function updateSnapshotState(originalSnapshotState, partialSnapshotState) {
-  if (global.UNSTABLE_SKIP_REPORTING) {
-    return originalSnapshotState;
-  }
-  return merge(originalSnapshotState, partialSnapshotState);
-}
-
 function checkResult({
   result,
   snapshotState,
   retryTimes,
+  currentTestName,
   snapshotIdentifier,
   chalk,
   dumpDiffToConsole,
@@ -59,18 +52,18 @@ function checkResult({
   if (result.updated) {
     // once transition away from jasmine is done this will be a lot more elegant and pure
     // https://github.com/facebook/jest/pull/3668
-    updateSnapshotState(snapshotState, { updated: snapshotState.updated + 1 });
+    snapshotState.updated.increment(currentTestName);
   } else if (result.added) {
-    updateSnapshotState(snapshotState, { added: snapshotState.added + 1 });
+    snapshotState.added.increment(currentTestName);
   } else {
     ({ pass } = result);
 
     if (pass) {
-      updateSnapshotState(snapshotState, { matched: snapshotState.matched + 1 });
+      snapshotState.matched.increment(currentTestName);
     } else {
       const currentRun = timesCalled.get(snapshotIdentifier);
       if (!retryTimes || (currentRun > retryTimes)) {
-        updateSnapshotState(snapshotState, { unmatched: snapshotState.unmatched + 1 });
+        snapshotState.unmatched.increment(currentTestName);
       }
 
       const differencePercentage = result.diffRatio * 100;
@@ -202,7 +195,7 @@ function configureToMatchImageSnapshot({
 
     if (isNot) { throw new Error('Jest: `.not` cannot be used with `.toMatchImageSnapshot()`.'); }
 
-    updateSnapshotState(snapshotState, { _counters: snapshotState._counters.set(currentTestName, (snapshotState._counters.get(currentTestName) || 0) + 1) }); // eslint-disable-line max-len
+    snapshotState._counters.increment(currentTestName);
 
     const snapshotIdentifier = createSnapshotIdentifier({
       retryTimes,
@@ -257,6 +250,7 @@ function configureToMatchImageSnapshot({
 
     return checkResult({
       result,
+      currentTestName,
       snapshotState,
       retryTimes,
       snapshotIdentifier,
@@ -271,5 +265,4 @@ function configureToMatchImageSnapshot({
 module.exports = {
   toMatchImageSnapshot: configureToMatchImageSnapshot(),
   configureToMatchImageSnapshot,
-  updateSnapshotState,
 };
