diff --git a/index.js b/index.js
index 4a5a0f0bd7459bce97a245faa740a669deca8a8b..e4df771d52889794fedfe16c755d451b4eae3051 100644
--- a/index.js
+++ b/index.js
@@ -1,5 +1,5 @@
 import Queue from 'yocto-queue';
-import {AsyncResource} from '#async_hooks';
+import {AsyncResource} from 'node:async_hooks';
 
 export default function pLimit(concurrency) {
 	if (!((Number.isInteger(concurrency) || concurrency === Number.POSITIVE_INFINITY) && concurrency > 0)) {
