diff --git a/dist/util/declaration-writer.js b/dist/util/declaration-writer.js
index 0e49be895aed56e234619b013d969691c26ff32c..c3f95bb196a92d83609cd4a3d364d1a20c5fb39a 100644
--- a/dist/util/declaration-writer.js
+++ b/dist/util/declaration-writer.js
@@ -23,7 +23,7 @@ class DeclarationWriter {
     async template() {
         if (this.multifile) {
             const ext = this.importFileExtension ? `.${this.importFileExtension}` : '';
-            return `${constants_1.MODIFIED_HEADER}\nimport type * as PJTG from '../pjtg${ext}';\n${this.content}`;
+            return `${constants_1.MODIFIED_HEADER}\n// @ts-nocheck\nimport type * as PJTG from '../pjtg${ext}';\n${this.content}`;
         }
         return `${constants_1.MODIFIED_HEADER}\n${await getNamespacePrelude(this.options.namespace)}\n${this.content}`;
     }
