diff --git a/dist/index.js b/dist/index.js
index 4f5d4edce3d0d84b98c8d7a962ebb03d706d9528..fd4864f1f50ada6352e9d6617ef7822a1ced16c2 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -176,20 +176,6 @@ var Batch = class {
     return __async(this, arguments, function* (payload, options = {}) {
       const emails = [];
       for (const email of payload) {
-        if (email.react) {
-          if (!this.renderAsync) {
-            try {
-              const { renderAsync } = yield import("@react-email/render");
-              this.renderAsync = renderAsync;
-            } catch (error) {
-              throw new Error(
-                "Failed to render React component. Make sure to install `@react-email/render`"
-              );
-            }
-          }
-          email.html = yield this.renderAsync(email.react);
-          email.react = void 0;
-        }
         emails.push(parseEmailToApiOptions(email));
       }
       const data = yield this.resend.post(
@@ -209,21 +195,6 @@ var Broadcasts = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
-        }
-        payload.html = yield this.renderAsync(
-          payload.react
-        );
-      }
       const data = yield this.resend.post(
         "/broadcasts",
         {
@@ -459,21 +430,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
-        }
-        payload.html = yield this.renderAsync(
-          payload.react
-        );
-      }
       const data = yield this.resend.post(
         "/emails",
         parseEmailToApiOptions(payload),
diff --git a/package.json b/package.json
index f98892471801e0bad018cbf9d19333e6953a45bd..59a61011a7fb3ef267022e2bc6a7042566894854 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "dependencies": {
-    "@react-email/render": "1.1.2"
-  },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
     "@types/jest": "29.5.14",
