diff --git a/dist/index.js b/dist/index.js
index 0a7d28841aa34a32b81004b490283b6eda285812..3f97ef703f694e8bf25fdbc93bd26e518f84db64 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -185,20 +185,6 @@ var Batch = class {
     return __async(this, arguments, function* (payload, options = {}) {
       const emails = [];
       for (const email of payload) {
-        if (email.react) {
-          if (!this.renderAsync) {
-            try {
-              const { renderAsync } = yield import("@react-email/render");
-              this.renderAsync = renderAsync;
-            } catch (error) {
-              throw new Error(
-                "Failed to render React component. Make sure to install `@react-email/render`"
-              );
-            }
-          }
-          email.html = yield this.renderAsync(email.react);
-          email.react = void 0;
-        }
         emails.push(parseEmailToApiOptions(email));
       }
       const data = yield this.resend.post(
@@ -218,21 +204,6 @@ var Broadcasts = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
-        }
-        payload.html = yield this.renderAsync(
-          payload.react
-        );
-      }
       const data = yield this.resend.post(
         "/broadcasts",
         {
@@ -468,21 +439,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
-        }
-        payload.html = yield this.renderAsync(
-          payload.react
-        );
-      }
       const data = yield this.resend.post(
         "/emails",
         parseEmailToApiOptions(payload),
diff --git a/package.json b/package.json
index 574b37a24a64443c85d4b700a4a434899ba364c8..4cafe3b82aefa12df7014296514a24ad88802fe6 100644
--- a/package.json
+++ b/package.json
@@ -41,14 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "peerDependencies": {
-    "@react-email/render": "^1.1.0"
-  },
-  "peerDependenciesMeta": {
-    "@react-email/render": {
-      "optional": true
-    }
-  },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
     "@types/jest": "29.5.14",
