diff --git a/dist/index.js b/dist/index.js
index a7061681a5b5f69961780b4c9d92d33ab485f415..6c8e1bd3d2eed33ed00687082de18dbc3b7574c2 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -159,14 +159,9 @@ var Batch = class {
       for (const email of payload) {
         if (email.react) {
           if (!this.renderAsync) {
-            try {
-              const { renderAsync } = yield import("@react-email/render");
-              this.renderAsync = renderAsync;
-            } catch (error) {
-              throw new Error(
-                "Failed to render React component. Make sure to install `@react-email/render`"
-              );
-            }
+            throw new Error(
+              "Failed to render React component. Make sure to install `@react-email/render`"
+            );
           }
           email.html = yield this.renderAsync(email.react);
           email.react = void 0;
@@ -191,14 +186,9 @@ var Broadcasts = class {
     return __async(this, arguments, function* (payload, options = {}) {
       if (payload.react) {
         if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
+          throw new Error(
+            "Failed to render React component. Make sure to install `@react-email/render`"
+          );
         }
         payload.html = yield this.renderAsync(
           payload.react
@@ -405,14 +395,9 @@ var Emails = class {
     return __async(this, arguments, function* (payload, options = {}) {
       if (payload.react) {
         if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
+          throw new Error(
+            "Failed to render React component. Make sure to install `@react-email/render`"
+          );
         }
         payload.html = yield this.renderAsync(
           payload.react
diff --git a/package.json b/package.json
index f0ade031dfcdc572f983935254eb191dcabbeac4..2e21dca55761bed7105018e3cb8bf539065e9f94 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "dependencies": {
-    "@react-email/render": "1.0.1"
-  },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
     "@types/jest": "29.5.12",
