diff --git a/dist/index.js b/dist/index.js
index 60e9daecc338cf028d988596adaed54337ae4525..24c66e02d3f8e4398767a02be97017ca5cfcb14b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -135,7 +135,6 @@ var Audiences = class {
 };
 
 // src/batch/batch.ts
-var import_render = require("@react-email/render");
 var Batch = class {
   constructor(resend) {
     this.resend = resend;
@@ -147,12 +146,6 @@ var Batch = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      for (const email of payload) {
-        if (email.react) {
-          email.html = yield (0, import_render.renderAsync)(email.react);
-          delete email.react;
-        }
-      }
       const data = yield this.resend.post(
         "/emails/batch",
         payload,
@@ -283,7 +276,6 @@ var Domains = class {
 };
 
 // src/emails/emails.ts
-var import_render2 = require("@react-email/render");
 var Emails = class {
   constructor(resend) {
     this.resend = resend;
@@ -295,10 +287,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        payload.html = yield (0, import_render2.renderAsync)(payload.react);
-        delete payload.react;
-      }
       const data = yield this.resend.post(
         "/emails",
         payload,
diff --git a/package.json b/package.json
index b95778591d143d8c917fdda64ce68c9fe6fc47df..140a98367604d4be4ffa56017e2a8ecca93f3f4d 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "dependencies": {
-    "@react-email/render": "0.0.16"
-  },
   "devDependencies": {
     "@biomejs/biome": "1.8.1",
     "@types/jest": "29.5.12",
