diff --git a/dist/index.js b/dist/index.js
index 81b26f28a50dc05167550147a4f68a32c7b9f3c3..5cc497ee104530890be491ffeb9a4613ec740d35 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -176,20 +176,6 @@ var Batch = class {
     return __async(this, arguments, function* (payload, options = {}) {
       const emails = [];
       for (const email of payload) {
-        if (email.react) {
-          if (!this.renderAsync) {
-            try {
-              const { renderAsync } = yield import("@react-email/render");
-              this.renderAsync = renderAsync;
-            } catch (error) {
-              throw new Error(
-                "Failed to render React component. Make sure to install `@react-email/render`"
-              );
-            }
-          }
-          email.html = yield this.renderAsync(email.react);
-          email.react = void 0;
-        }
         emails.push(parseEmailToApiOptions(email));
       }
       const data = yield this.resend.post(
@@ -209,21 +195,6 @@ var Broadcasts = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
-        }
-        payload.html = yield this.renderAsync(
-          payload.react
-        );
-      }
       const data = yield this.resend.post(
         "/broadcasts",
         {
@@ -441,21 +412,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        if (!this.renderAsync) {
-          try {
-            const { renderAsync } = yield import("@react-email/render");
-            this.renderAsync = renderAsync;
-          } catch (error) {
-            throw new Error(
-              "Failed to render React component. Make sure to install `@react-email/render`"
-            );
-          }
-        }
-        payload.html = yield this.renderAsync(
-          payload.react
-        );
-      }
       const data = yield this.resend.post(
         "/emails",
         parseEmailToApiOptions(payload),
diff --git a/package.json b/package.json
index f9419bbb0bc8544cad1ace816032d7f56cb9cfae..d34682f12b4c86e486e46f6c5b02284912250bc9 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "dependencies": {
-    "@react-email/render": "1.0.5"
-  },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
     "@types/jest": "29.5.12",
