diff --git a/dist/index.js b/dist/index.js
index d0ed6fc1187c455ad9136fc4fda2ea21bc2da9bf..d02e22fe4d4a9a31ec0cf281b1bef8196a8e987d 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -135,7 +135,6 @@ var Audiences = class {
 };
 
 // src/batch/batch.ts
-var import_render = require("@react-email/render");
 var Batch = class {
   constructor(resend) {
     this.resend = resend;
@@ -147,12 +146,6 @@ var Batch = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      for (const email of payload) {
-        if (email.react) {
-          email.html = yield (0, import_render.renderAsync)(email.react);
-          delete email.react;
-        }
-      }
       const data = yield this.resend.post(
         "/emails/batch",
         payload,
@@ -283,7 +276,6 @@ var Domains = class {
 };
 
 // src/emails/emails.ts
-var import_render2 = require("@react-email/render");
 var Emails = class {
   constructor(resend) {
     this.resend = resend;
@@ -295,10 +287,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        payload.html = yield (0, import_render2.renderAsync)(payload.react);
-        delete payload.react;
-      }
       const data = yield this.resend.post(
         "/emails",
         payload,
diff --git a/dist/index.mjs b/dist/index.mjs
index 2ef41bd7e252c60b1b2e5a82e8898b7f74b66809..e589b29483f4f39a5a82d31d2e6b695a568be982 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -112,7 +112,6 @@ var Audiences = class {
 };
 
 // src/batch/batch.ts
-import { renderAsync } from "@react-email/render";
 var Batch = class {
   constructor(resend) {
     this.resend = resend;
@@ -124,12 +123,6 @@ var Batch = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      for (const email of payload) {
-        if (email.react) {
-          email.html = yield renderAsync(email.react);
-          delete email.react;
-        }
-      }
       const data = yield this.resend.post(
         "/emails/batch",
         payload,
@@ -260,7 +253,6 @@ var Domains = class {
 };
 
 // src/emails/emails.ts
-import { renderAsync as renderAsync2 } from "@react-email/render";
 var Emails = class {
   constructor(resend) {
     this.resend = resend;
@@ -272,10 +264,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        payload.html = yield renderAsync2(payload.react);
-        delete payload.react;
-      }
       const data = yield this.resend.post(
         "/emails",
         payload,
diff --git a/package.json b/package.json
index 3625d776eef78953dfc85bc2e1ff0a949f7c4632..f94e9bd3d79696f26432840f7e04affca294dc8d 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "dependencies": {
-    "@react-email/render": "0.0.15"
-  },
   "devDependencies": {
     "@biomejs/biome": "1.8.1",
     "@types/jest": "29.5.12",
