diff --git a/dist/index.js b/dist/index.js
index 6dc469d499266c0c6e0ec02d0682396eeca6a984..de32109eb72e92221880604ec7736ad8553cecb7 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -135,7 +135,6 @@ var Audiences = class {
 };
 
 // src/batch/batch.ts
-var import_render = require("@react-email/render");
 var Batch = class {
   constructor(resend) {
     this.resend = resend;
@@ -147,12 +146,6 @@ var Batch = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      for (const email of payload) {
-        if (email.react) {
-          email.html = yield (0, import_render.renderAsync)(email.react);
-          delete email.react;
-        }
-      }
       const data = yield this.resend.post(
         "/emails/batch",
         payload,
@@ -283,7 +276,6 @@ var Domains = class {
 };
 
 // src/emails/emails.ts
-var import_render2 = require("@react-email/render");
 var Emails = class {
   constructor(resend) {
     this.resend = resend;
@@ -295,10 +287,6 @@ var Emails = class {
   }
   create(_0) {
     return __async(this, arguments, function* (payload, options = {}) {
-      if (payload.react) {
-        payload.html = yield (0, import_render2.renderAsync)(payload.react);
-        delete payload.react;
-      }
       const data = yield this.resend.post(
         "/emails",
         {
diff --git a/package.json b/package.json
index fe39c8a6fa6cfc3805c870bbb04bceec0e0619fd..eaf5e7450beaf6ff081fe3fb79bb736f0db283f5 100644
--- a/package.json
+++ b/package.json
@@ -41,9 +41,6 @@
     "url": "https://github.com/resendlabs/resend-node/issues"
   },
   "homepage": "https://github.com/resendlabs/resend-node#readme",
-  "dependencies": {
-    "@react-email/render": "0.0.17"
-  },
   "devDependencies": {
     "@biomejs/biome": "1.8.3",
     "@types/jest": "29.5.12",
