diff --git a/lib/lib.dom.d.ts b/lib/lib.dom.d.ts
index edf900a5de22fd63305f87e32d5974716eda7ddd..20915d0afa1dbd993802e7f3bf4f246fe55609fc 100644
--- a/lib/lib.dom.d.ts
+++ b/lib/lib.dom.d.ts
@@ -26972,8 +26972,6 @@ interface Window extends EventTarget, AnimationFrameProvider, GlobalEventHandler
      * [MDN Reference](https://developer.mozilla.org/docs/Web/API/Window/menubar)
      */
     readonly menubar: BarProp;
-    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Window/name) */
-    name: string;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Window/navigator) */
     readonly navigator: Navigator;
     /**
diff --git a/lib/lib.webworker.d.ts b/lib/lib.webworker.d.ts
index dd6a69e7643aabf716ee383c967bd2ecc9f8cd11..9eb59a16b8ecb03cd722183e0f0abcdb4061b0bd 100644
--- a/lib/lib.webworker.d.ts
+++ b/lib/lib.webworker.d.ts
@@ -2518,12 +2518,6 @@ interface DedicatedWorkerGlobalScopeEventMap extends WorkerGlobalScopeEventMap,
  * [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope)
  */
 interface DedicatedWorkerGlobalScope extends WorkerGlobalScope, AnimationFrameProvider, MessageEventTarget<DedicatedWorkerGlobalScope> {
-    /**
-     * Returns dedicatedWorkerGlobal's name, i.e. the value given to the Worker constructor. Primarily useful for debugging.
-     *
-     * [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/name)
-     */
-    readonly name: string;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/rtctransform_event) */
     onrtctransform: ((this: DedicatedWorkerGlobalScope, ev: RTCTransformEvent) => any) | null;
     /**
