diff --git a/lib/lib.dom.d.ts b/lib/lib.dom.d.ts
index 14508e8dc3effb02a4bd808e61770aacbc8b13b7..3ba9862acef202d352fbd227bed3573af4714a76 100644
--- a/lib/lib.dom.d.ts
+++ b/lib/lib.dom.d.ts
@@ -36461,12 +36461,6 @@ interface Window extends EventTarget, AnimationFrameProvider, GlobalEventHandler
      * [MDN Reference](https://developer.mozilla.org/docs/Web/API/Window/menubar)
      */
     readonly menubar: BarProp;
-    /**
-     * The `Window.name` property gets/sets the name of the window's browsing context.
-     *
-     * [MDN Reference](https://developer.mozilla.org/docs/Web/API/Window/name)
-     */
-    name: string;
     /**
      * The **`Window.navigator`** read-only property returns a reference to the Navigator object, which has methods and properties about the application running the script.
      *
diff --git a/lib/lib.webworker.d.ts b/lib/lib.webworker.d.ts
index 12f4460b8d7f5f4930779e24b1dae10138de3796..dc0075786859d6dc980e1867ad56bbf9f84c9dd6 100644
--- a/lib/lib.webworker.d.ts
+++ b/lib/lib.webworker.d.ts
@@ -3520,12 +3520,6 @@ interface DedicatedWorkerGlobalScopeEventMap extends WorkerGlobalScopeEventMap,
  * [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope)
  */
 interface DedicatedWorkerGlobalScope extends WorkerGlobalScope, AnimationFrameProvider, MessageEventTarget<DedicatedWorkerGlobalScope> {
-    /**
-     * The **`name`** read-only property of the the Worker.Worker constructor can pass to get a reference to the DedicatedWorkerGlobalScope.
-     *
-     * [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/name)
-     */
-    readonly name: string;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/DedicatedWorkerGlobalScope/rtctransform_event) */
     onrtctransform: ((this: DedicatedWorkerGlobalScope, ev: RTCTransformEvent) => any) | null;
     /**
